hazelcast-spring/src/main/java/com/hazelcast/spring/HazelcastConfigBeanDefinitionParser.java

```diff
@@ -593,6 +644,22 @@ public void handleQueueStoreConfig(Node node, BeanDefinitionBuilder queueConfigB
             queueConfigBuilder.addPropertyValue("queueStoreConfig", beanDefinition);
         }
 
+        private BeanDefinitionBuilder extractBasicStoreConfig(Node node,
+                                                              BeanDefinitionBuilder builder) {
+            final String implAttrName = "store-implementation";
```

Comment:
Changed. Also changed the queue store variable name since it is analogous to the ring buffer store config.